ChangeLog

```diff
@@ -1,6 +1,9 @@
 Version 10.2.0
+-------------
     * Add timeouts to Elan's remote client
     * Add metrics to Elan's remote client
+	* Only return delete errors in Elan's gc
+	* Fix purity clean
```

Comment:
I think you have to make a new version for the release pipeline to be triggered.

Reply:
It looks like the release pipeline didn't produce any assets anyway so I was going to delete it before landing this

Reply:
But still, are you sure it's gonna get triggered when merging this? I thought it was getting triggered if the `VERSION` is changed

Reply:
Looks like the release _attempted_ to trigger, but failed: https://github.com/thought-machine/please-servers/actions/runs/3311956028/jobs/5468074324#step:6:30

Reply:
The release process is pretty flaky. I'm sure I've seen that error before!

I have bumped the version in this PR